src/OneSignalMessage.php

```diff
@@ -3,32 +3,20 @@
 namespace NotificationChannels\OneSignal;
 
 use Illuminate\Support\Arr;
+use NotificationChannels\OneSignal\Traits\Categories\AppearanceHelpers;
+use NotificationChannels\OneSignal\Traits\Categories\AttachmentHelpers;
+use NotificationChannels\OneSignal\Traits\Categories\ButtonHelpers;
+use NotificationChannels\OneSignal\Traits\Categories\DeliveryHelpers;
+use NotificationChannels\OneSignal\Traits\Categories\GroupingHelpers;
+
 
 class OneSignalMessage
 {
-    /** @var string */
-    protected $body;
-
-    /** @var string */
-    protected $subject;
-
-    /** @var string */
-    protected $url;
-
-    /** @var string */
-    protected $icon;
-
-    /** @var array */
-    protected $data = [];
 
-    /** @var array */
-    protected $buttons = [];
+    use AppearanceHelpers, AttachmentHelpers, ButtonHelpers, DeliveryHelpers, GroupingHelpers;
```

Comment:
You could use relative trait names here - then you could remove all of the top `use` imports.

i.e.

```php
class OneSignalMessage
{
    use Traits\Categories\ButtonHelpers,
        Traits\Categories\GroupingHelpers,
        Traits\Categories\DeliveryHelpers,
        Traits\Categories\AppearanceHelpers,
        Traits\Categories\AttachmentHelpers;
```

But that is just a style thing - and not really needed